### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by Sekgomotso**

### CODE REVIEW

File: acc/templates/acc/form.html [hunk 1]
```diff
@@ -34,6 +35,15 @@ <h5 class="card-header text-black text-center py-4">
...
+            <!-- success message -->
+            {% if messages %}
+            <ul class="messages">
+                {% for message in messages %}
+                <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+                {% endfor %}
+            </ul>
+            {% endif %}
+
             <form class="custom-control" id="contact" method="post" action="{% url 'home_page' %}">
```
Comment (Sjk-Lacoste) [inline acc/templates/acc/form.html:47]: Greate but it can still work even if you don't include action attribute

File: acc/views.py [hunk 2]
```diff
@@ -10,13 +11,21 @@ def home_page(request):
...
         name = request.POST["fname"]
         surname = request.POST["lname"]
+        email = request.POST["email"]
         career = request.POST["career"]
         bio = request.POST["about"]
 
+        # email details
         send_mail('Registration form', 
-         'First name: ' + name + '\n' + 'Last name: ' + surname + '\n' + 'Choice: ' + career + '\n' + 'About user: ' + bio,
+         'First name: ' + name + '\n' + 'Last name: ' + surname + '\n' + 'User email: ' + email + '\n' 'Choice: ' + career + '\n' + 'About user: ' + bio,
```
Comment (Sjk-Lacoste) [inline acc/views.py:20]: Next time use f-sting, it is much easier to read and its fast
`print(f'First name: {name} \n Last Name: {surname}')`

File: acc/templates/acc/form.html (hunk 1 above)
Comment (Sjk-Lacoste) [inline acc/templates/acc/form.html:47]: Even if you don't include action it would still work

File: acc/views.py (hunk 2 above)
Comment (Sekgomotso) [reply acc/views.py:20]: Got it!

**Pull request merged**